src/main/java/com/coyoapp/tinytask/service/DefaultTaskService.java

```diff
@@ -20,13 +23,16 @@
 public class DefaultTaskService implements TaskService {
 
   private final TaskRepository taskRepository;
+  private final FileService fileService;
   private final MapperFacade mapperFacade;
 
   @Override
   @Transactional
```

Comment:
optionals as parameter is generally not the best idea. i ask you to do some research on this yourself, so that I can skip the argumentation here.

Reply:
It depends on project coding conventions. There is no blocker argument not to use that.